FinSearchUnified/BusinessLogic/Models/FindologicArticleModel.php

```diff
@@ -837,4 +839,16 @@ public function getXmlRepresentation()
     {
         return $this->xmlArticle;
     }
+
+    private function isCrossSellingCategoryConfigured()
```

Comment:
Looking at this method makes me think it returns a boolean. Make this return a boolean and set `$this->shouldBeExported` to the return value of this method. Stop the execution in the constructor by `return`ing if this method returns `true` (product has a cross sell category).

Reply:
OK, in that case ignore my previous comment.